package.json

```diff
@@ -30,7 +30,11 @@
         "onLanguage:objective-cpp",
         "onCommand:clangd.activate",
         "onCommand:clangd.install",
-        "onCommand:clangd.update"
+        "onCommand:clangd.update",
+        "onCommand:clangd.restart",
+        "onCommand:clangd.switchheadersource",
```

Comment:
`clangd.switchheadersource` and `clangd.restart` won't be really useful if clangd isn't available already. but i suppose it makes sense for `clangd.{project,user}config`.

looping in @hokein as he uses vscode and might have opinions about the matter.